go.mod

```diff
@@ -28,8 +31,12 @@ require (
 	k8s.io/api v0.24.3
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.3
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
+	stathat.com/c/consistent v1.0.0
 )
 
+require github.com/alicebob/miniredis/v2 v2.23.0
```

Comment:
Odd that this made its own group? 

`go mod tidy -compat=1.17` should resolve this into two groups.

Reply:
I'm doing a big version bump + cleanup (I have protobuf problems on my wip branch, I think it's just because the branch is super old though), and it cleans this up.